src/flows/pipes/Downsample/view.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {FC, useContext} from 'react'
+import {
+  FlexBox,
+  FlexDirection,
+  ComponentSize,
+  InputLabel,
+  IconFont,
+  ComponentColor,
+  Toggle,
+  InputToggleType,
+  QuestionMarkTooltip,
+  ComponentStatus,
+  MultiSelectDropdown,
+} from '@influxdata/clockface'
+
+import {DURATIONS, FUNCTIONS} from 'src/timeMachine/constants/queryBuilder'
+
+import {PipeContext} from 'src/flows/context/pipe'
+import DurationInput from 'src/shared/components/DurationInput'
+
+import {PipeProp} from 'src/types/flows'
+
+const Downsample: FC<PipeProp> = ({Context}) => {
+  const {data, update} = useContext(PipeContext)
+
+  const selectFn = fn => {
+    const fns = [...data.functions]
+    const fnsFound = fns.map(f => f.name).indexOf(fn)
+    if (fnsFound !== -1) {
+      fns.splice(fnsFound, 1)
+      update({
+        functions: fns,
+      })
+    } else {
+      update({
+        functions: [...fns, {name: fn}],
+      })
+    }
+  }
+
+  const setPeriod = _period => {
+    update({
+      aggregateWindow: {
+        ...data.aggregateWindow,
+        period: _period,
+      },
+    })
+  }
+
+  const toggleFill = () => {
+    update({
+      aggregateWindow: {
+        ...data.aggregateWindow,
+        fillValues: !data.aggregateWindow.fillValues,
+      },
+    })
+  }
+
+  return (
+    <Context>
+      <div className="downsample-panel">
+        <div className="downsample-panel--left">
+          <h5>Apply aggregate</h5>
+          <MultiSelectDropdown
+            emptyText="Select"
+            style={{width: '250px'}}
+            options={FUNCTIONS.map(f => f.name)}
+            selectedOptions={data.functions.map(fn => fn.name)}
+            onSelect={selectFn}
+            buttonColor={ComponentColor.Secondary}
+            buttonIcon={IconFont.BarChart}
+          />
+        </div>
+        <div className="downsample-panel--right">
+          <h5>Window Period</h5>
```

Comment:
Input label should say "Every Window Period"? Trying to connect the concept of "Apply Aggregate" to "Every Window Period"